contrib/mounts.conf

```diff
@@ -0,0 +1 @@
+/usr/share/rhel/secrets:/run/secrets
```

Comment:
Lets create a subdir called RHEL and put this file there and then create an empty one for others to use.

Reply:
I have moved this mounts.conf into rhel. If you can point me to other distro examples happy to include them in this PR.